### PR TITLE
[IMP] digest: milkify the digest email template

### PR DESCRIPTION
**Pull request by edi-odoo**
This PR contains a revamp of the digest email in the new "Milk" style replacing
the old purple with the new one. This has been adapted in all digest data too.

The images of the digest email's tips have been replaced by "milkified"
versions of them, already available on odoo cdn by the way.

Task-3338467

### CODE REVIEW

**Comment by edi-odoo** [comment]
@tde-banana-odoo could you review please ? :)

**Comment by edi-odoo** [comment]
👋 @JKE-be 
I've replaced the images in the repo in this PR and this one https://github.com/odoo/enterprise/pull/42790
Only one isn't currently used anywhere since we don't have any content yet (`milk-ctrl+k.gif`), and the `odoo-mobile.png` has been uploaded via odoo.com as the older version was.

Thanks!

**Comment by edi-odoo** [comment]
Style check to be ignored, the gif is compressed at its maximum (more compression would result in more colors being lost)

File: addons/website/data/digest_data.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <div>
     <p class="tip_title">Tip: Engage with visitors to convert them into leads</p>
     <p class="tip_content">Monitor your visitors while they are browsing your website with the Odoo Social app. Engage with them in just a click using a live chat request or a push notification. If they have completed one of your forms, you can send them an SMS, or call them right away while they are browsing your website.</p>
-    <img src="https://download.odoocdn.com/digests/website/static/src/img/website-visitors.gif" width="540" class="illustration_border" />
```
[the remark this answers is not in the document]
Comment (tde-banana-odoo) [reply addons/website/data/digest_data.xml:12]: Hello, we want to keep names unique indeed, as they target different versions. That way we can keep a v15 oriented gif for people using tips in v15 for example.

[the remark this answers is not in the document]
Comment (rdeodoo) [reply addons/website/data/digest_data.xml:12]: Hm I see, I think you already explained that to me once :grimacing: , thanks again :)

[the remark this answers is not in the document]
Comment (tde-banana-odoo) [reply addons/website/data/digest_data.xml:12]: :D

**Review by Bertrand2 — APPROVED** [review]

**Comment by edi-odoo** [comment]
@tde-banana-odoo this has been forgotten, could you review please? Thanks !

**Review by qdp-odoo — APPROVED** [review]

**Comment by tde-banana-odoo** [comment]
@robodoo override=ci/style

Reason: well, 108 Kb instead of 100 is reasonable, especially all gifs have the same size / quality. Would be a shame to alter those to gain 8 Kb. Let us hope we don't go for chocolate design soonish.

**Comment by tde-banana-odoo** [comment]
@robodoo override=ci/security

Reason: Markup was already present, just changing its content. And anyway dynamic content is escaped, outside of Markup tag.

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-merge

**Pull request closed without merging**